mongodb-crypt/build.gradle.kts

```diff
@@ -175,7 +175,7 @@ afterEvaluate {
     tasks.jar {
         manifest {
             attributes(
-                "-exportcontents" to "com.mongodb.crypt.capi.*;-noimport:=true",
+                "-exportcontents" to "com.mongodb.*;-noimport:=true",
```

Comment:
This is a weird one.  driver-core uses `*` for this, but `*` causes a warning with mongodb-crypt because the jar file contains directories for each of the shared library files, and some of those (e.g. `linux-aarch64`) are not valid Java package names.  Using `com.mongodb.*` does the trick (it could also just be `com.*`).